### ISSUE TITLE
Remove fancy drupal core version lookup

### ISSUE THREAD

**Issue by grayside** [phase2/generator-gadget#90]
In master right now is a overly complicated system. Let's uncomplicate it.

1. The Drupal 8 composer template has drupal/core: true as opposed to a real semver range or version.
2. Elsewhere, we are detecting that value is present, and performing a lookup of the Drupal.org update system to find the latest stable minor release of D8.
3. This is then injected into the template.
4. That value is then used by another not-quite-open generator to make decisions about things like docker images to use.

Let's just change the value to ^8.3 or the like, remove that particular spot of cleverness from this generator doing the lookup, make sure Octane successfully removes drupal/core from it's template, and then test that this generator and the other generator both work.

**Comment by mike-potter** [comment]
Agreed, although "^8.0" might make the intention even clearer so people don't assume they need to change it to "^8.4" when that comes out.
